### PR TITLE
Device list inverter support

### PR DESCRIPTION
**Pull request by blammit**

### CODE REVIEW

File: pages/settings/devicelist/inverter/PageInverter.qml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+			ListQuantityItem {
+				//% "System yield"
+				text: qsTrId("inverter_system_yield")
+				visible: defaultVisible && dataItem.isValid
+				unit: VenusOS.Units_Energy_KiloWattHour
+				dataItem.uid: root.bindPrefix + "/Yield/System"
+			}
+
+			ListQuantityItem {
+				//% "State of charge"
```
Comment (DanielMcInnes) [inline pages/settings/devicelist/inverter/PageInverter.qml:172]: CommonWords?

**Review by DanielMcInnes — COMMENTED** [review]
I noticed something broken with this drop down box, I have to click it twice to get it to open. The first click doesn't work. Wasm build.

Screenshot from 2024-01-15 09-43-07

**Review by DanielMcInnes — COMMENTED** [review]
Screenshot from 2024-01-15 09-53-29
If I click on the pink 'X' here to close the daily yield, the app crashes (wasm only)

File: components/settings/AlarmLevelRadioButtonGroup.qml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import QtQuick
+import Victron.VenusOS
+
+ListRadioButtonGroup {
+	optionModel: [
+		{ display: CommonWords.disabled, value: 0 },
+		//% "Alarm only"
+		{ display: qsTrId("alarm_level_alarm_only"), value: 1 }, // no pre-alarms
+		//% "Alarms & warnings"
+		{ display: qsTrId("alarm_level_p00alarms_and_warnings"), value: 2 }
```
Comment (DanielMcInnes) [inline components/settings/AlarmLevelRadioButtonGroup.qml:15]: 'p00' ?

Comment (DanielMcInnes) [reply components/settings/AlarmLevelRadioButtonGroup.qml:15]: also, both of these strings can go in CommonWords

File: pages/settings/devicelist/inverter/PageInverter.qml [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+
+	GradientListView {
+		model: ObjectModel {
+
+			ListRadioButtonGroup {
+				id: modeSwitch
+
+				//: Change the inverter mode
+				//% "Switch"
+				text: qsTrId("inverter_switch")
```
Comment (DanielMcInnes) [inline pages/settings/devicelist/inverter/PageInverter.qml:49]: CommonWords?

File: pages/settings/devicelist/inverter/PageInverter.qml (hunk 1 above)
Comment (blammit) [reply pages/settings/devicelist/inverter/PageInverter.qml]: done

File: components/settings/AlarmLevelRadioButtonGroup.qml (hunk 2 above)
Comment (blammit) [reply components/settings/AlarmLevelRadioButtonGroup.qml]: Hahah oops, fixed

File: pages/settings/devicelist/inverter/PageInverter.qml (hunk 3 above)
Comment (blammit) [reply pages/settings/devicelist/inverter/PageInverter.qml]: done

**Comment by blammit** [comment]
> If I click on the pink 'X' here to close the daily yield, the app crashes (wasm only)

Thanks for the catch -- I think we'll need to do something to make the dialogs modal. This needs more investigation, will make a separate task for it.

> I noticed something broken with this drop down box, I have to click it twice to get it to open. The first click doesn't work. Wasm build.

Fixed this, filed QTBUG-121029.

**Review by DanielMcInnes — APPROVED** [review]
lgtm

**Pull request merged**